config/schema/document_types/asylum_support_decision.json

```diff
@@ -0,0 +1,190 @@
+{
+  "fields": [
+    "tribunal_decision_decision_date",
+    "tribunal_decision_judges",
+    "tribunal_decision_category",
+    "tribunal_decision_sub_category",
+    "tribunal_decision_landmark",
+    "tribunal_decision_reference_number"
+  ],
+  "allowed_values": {
+    "tribunal_decision_judges": [
```

Comment:
Proposal how to avoid having these values in rummager here: https://github.com/alphagov/specialist-publisher/pull/542/files#r38083401